src/main/java/org/apache/maven/archiver/PomPropertiesUtil.java

```diff
@@ -71,8 +71,8 @@ private void createPropertiesFile(Properties properties, Path outputFile, boolea
             return;
         }
 
-        try (PrintWriter pw = new PrintWriter(outputFile.toFile(), StandardCharsets.ISO_8859_1.name());
-                StringWriter sw = new StringWriter()) {
+        try ( Writer writer = Files.newBufferedWriter(outputFile, StandardCharsets.ISO_8859_1);
```

Comment:
The `loadPropertiesFile` method let the `Properties` class decide the charset used (so it's using ISO_8859_1, as it's the default).  I think we should use an `OutputStream` here and pass it to `properties.store()` without the charset.

Reply:
good idea. That makes this much simpler.